site_scons/compiler_setup.py

```diff
@@ -79,9 +79,6 @@ def base_setup(env, prereqs=None):
     if config.CheckHeader('stdatomic.h'):
         config.Finish()
         env.AppendUnique(CPPDEFINES={'HAVE_STDATOMIC': '1'})
-    elif prereqs:
-        config.Finish()
-        prereqs.require(env, 'openpa', headers_only=True)
     else:
         config.Finish()
```

Comment:
You could tidy this code up slightly now by calling config.Finish() outside the conditional, it's not worth another push however.

The issue before was that if you called prereqs.require() with config still open the build would fail as you can only have one open config object per environment.

Reply:
oh right well I can do that